package.json

```diff
@@ -76,6 +76,7 @@
     "@ava/typescript": "^1.1.1",
     "@babel/core": "^7.9.6",
     "@octokit/graphql": "2.1.1",
+    "@tsconfig/node16-strictest": "^1.0.4",
```

Comment:
Based on [this comment](https://github.com/nasa/cumulus/pull/3228#issuecomment-1405233837), this is not needed and can be removed.